fili-core/src/main/java/com/yahoo/bard/webservice/web/filters/BardLoggingFilter.java

```diff
@@ -66,13 +65,13 @@
     ClientResponseFilter, WriterInterceptor {
 
     private static final Logger LOG = LoggerFactory.getLogger(BardLoggingFilter.class);
-    private static final String PROPERTY_NANOS = LoggingFilter.class.getName() + ".nanos";
```

Comment:
Since this class is now basically the `FiliTimingFilter++`, does it make sense for this filter to either have a `FiliTimingFilter` or be a `FiliTimingFilter`?

That would reduce duplicate code, as well as make it clearer that the `BardLoggingFilter` is doing the same thing as the `FiliTimingFilter` plus more.